erpnext/public/js/controllers/stock_controller.js

```diff
@@ -195,7 +195,7 @@ erpnext.stock.StockController = class StockController extends frappe.ui.form.Con
 							});
 						}
 					},
-					get_query: () => erpnext.queries.project_template(dialog.get_value('applies_to_item')),
+					get_query: () => erpnext.queries.project_template(dialog.get_values()),
```

Comment:
Pass only applies_to_item and determine Item Group and Vehicle region from the Item instead